lib/SIL/SILPrinter.cpp

```diff
@@ -1176,6 +1176,78 @@ class SILPrinter : public SILInstructionVisitor<SILPrinter> {
     *this << ") : ";
     *this << BI->getType();
   }
+
+  // SWIFT_ENABLE_TENSORFLOW
+  void visitSymbolicValue(SymbolicValue v) {
+    switch (v.getKind()) {
+    case swift::SymbolicValue::Integer:
```

Comment:
I don't think the swift:: namespaces are required.

Reply:
Done, sorry about that.
`swift::` was produced by Xcode when it auto-generated enum cases.